domain/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .type import *
 from .entity import *
 
+
```

Comment:
Remove the addition of this line. 

Remember to double check the changes you make, we want to keep the changes relevant to the issue they're fixing.